mycore.org/content/de/documentation/getting_started/gs_jpa.html

```diff
@@ -2,194 +2,404 @@
+            </tr>
+            <tr>
+                <td>hibernate.hikari.leakDetectionThreshold</td>
+                <td>MCR.JPA.Connection.LeakDetectionThreshold</td>
+            </tr>
+            <tr>
+                <td>hibernate.hikari.registerMbeans</td>
+                <td>MCR.JPA.Connection.RegisterMbeans</td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+
+<p>
+    Die Klasse MCRSimpleConfigPersistenceUnitDescriptor erweitert MCRPersistenceUnitDescriptor und bietet einfachere
+    Alternativen zur Konfiguration der Properties, um die Länge der Properties zu reduzieren und die Namenskonventionen
+    von MyCoRe einzuhalten.
+</p>
+
+<h3>Datenbank-Treiber</h3>
+<p>
+    Der für den Zugriff auf die Datenbank notwendige JDBC-Treiber (i.d.R. eine JAR-Datei) kann im <code>lib</code>
+    Ordner des Konfigurationsverzeichnisses abgelegt werden.
+    Alternativ kann der Treiber auch in der <code>pom.xml</code> der Anwendung als Dependency hinzugefügt werden. Dies
+    funktioniert jedoch nur, wenn die Anwendung als WAR-Datei verpackt wird.
+</p>
+
+
+<h3>Hinweis bei Performance-Problemen</h3>
+
+<p>Sollten in der Anwendung vermehrt Meldungen wie <code>Connection leak detection triggered</code> auftauchen, dann ist
+    das ein Hinweis, dass Verbindungen nicht rechtzeitig geschlossen werden. Das kann daran liegen, dass ein
+    langwieriger Prozess läuft. In diesem Fall kann die <code>hibernate.hikari.leakDetectionThreshold</code> erhöht werden.
+    HikariCP überprüft in regelmäßigen Abständen, ob eine Verbindung länger als die angegebene Zeit offen ist. Ist das
+    der Fall, wird eine Warnung ausgegeben. Der Standardwert beträgt 2 Sekunden. Der Wert wird in Millisekunden angegeben.
+    Sollte der Wert auf 0 gesetzt werden, wird die Überprüfung deaktiviert.
+    Sollte die Meldung jedoch auch bei kurzen Prozessen auftreten, dann sollte die Anwendung genauer untersucht werden,
+    um die Ursache zu finden. Es handelt sich in der Regel um ein Problem in der Anwendung, das behoben werden muss.
+    Es ist nicht empfehlenswert, die Verbindungen automatisch durch den Pool schließen zu lassen.
+    Die Connections sollten nicht automatisch durch den Pool geschlossen werden.
+</p>
+
+<blockquote class="blockquote text-center">
+    <p class="small">
+        HikariCP does not, in principle, believe that the pool should coverup connection leaks. We disagree with the
+        reasoning behind C3P0's unreturned connection handling.
+        The leak detection is there to help you pinpoint the source of the leak. That should provide enough clues to track
+        down the configuration or coding error.
+    </p>
+    <footer class="blockquote-footer"><a href="https://github.com/brettwooldridge/HikariCP/issues/612#issuecomment-209839908">Entwickler von HikariCP
+        auf Github</a></footer>
+</blockquote>
+
+{{< mcr-figure label="Konfiguration per persitence.xml" caption="<code>resources/META-INF/persistence.xml</code>">}}
```

Comment:
ein "s" in "persistence.xml" fehlt - ggf. nochmal in der ganzen Doku nach persitence" (ohne "s") suchen. Das habe ich, glaube ich, auch schon anderswo falsch sehen.